lib/models/finamp_models.dart

```diff
@@ -1315,7 +1315,7 @@ class DownloadItem extends DownloadStub {
 /// Enumerated by Isar, do not modify order or delete existing entries.
 enum DownloadItemType {
   collection(true, false),
-  track(true, true),
+  song(true, true),
```

Comment:
can you please add a warning here, not to rename this? preferably with the `//!!! <text>` syntax